### ISSUE TITLE
Mapping fields IV.1.1.1 and IV.1.1.2 in F25 and F23

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#260]
The following XPATH, belonging to fields IV.1.1.1 and IV.1.1.2 cannot be mapped, as there is no corresponding value for the procurement-procedure-type in EUVocabularies:
   `PT_AWARD_CONTRACT_WITH_PRIOR_PUBLICATION`
   `PT_AWARD_CONTRACT_WITHOUT_PUBLICATION`

See related issues in EPO and here: https://github.com/OP-TED/ePO/issues/424 and https://github.com/OP-TED/ted-rdf-mapping/issues/242

**Comment by muricna** [comment]
For F25 see reply in [255](https://github.com/OP-TED/ted-rdf-mapping/issues/255):

IV.1.1.1: A message has been sent to GROW, their interim solution is to map to procurement-procedure-type open.
IV.1.1.2: A message has been sent to GROW, their interim solution is to map to procurement-procedure-type neg-wo-call.

There is no order for F23 at this point so no comment on F23 for the time being - when the current deliverables have been accepted F23 can be addressed may it probably is similar to that of F25

**Comment by csnyulas** [comment]
Fixed in CM v6.6.1.

Based on our discussion with @muricna and @costezki in addition to mapping field IV.1.1.1 to `at-voc:procurement-procedure-type/open` and field IV.1.1.2 to `at-voc:procurement-procedure-type/neg-wo-call`, we agreed that it does make sense to keep the original mapping as well, for adding a second `rdf:type` (`epo:CompetitionNotice`) to the previous notice in case field IV.1.1.1 is checked.
We added explanation in the Notes column for all three mappings, to clarify the intent of the mappings, and have a reminder for updating the mappings when appropriate vocabulary terms are available.

Also fixed in technical mapping (see commit referring to this issue), so we can close this ticket.
